matron/src/screen_events.c

```diff
@@ -99,128 +98,120 @@ void* screen_event_loop(void* x) {
+                         ev->payload.d.d6);
+        break;
     case SCREEN_EVENT_ARC:
-	screen_arc(ev->payload.d.d1, ev->payload.d.d2,
-		   ev->payload.d.d3, ev->payload.d.d4,
-		   ev->payload.d.d5);
-	break;
+        screen_arc(ev->payload.d.d1, ev->payload.d.d2, ev->payload.d.d3, ev->payload.d.d4, ev->payload.d.d5);
+        break;
     case SCREEN_EVENT_RECT:
-	screen_rect(ev->payload.d.d1, ev->payload.d.d2,
-		    ev->payload.d.d3, ev->payload.d.d4);
-	break;
+        screen_rect(ev->payload.d.d1, ev->payload.d.d2, ev->payload.d.d3, ev->payload.d.d4);
+        break;
     case SCREEN_EVENT_STROKE:
-	screen_stroke();
-	break;
+        screen_stroke();
+        break;
     case SCREEN_EVENT_FILL:
-	screen_fill();
-	break;
+        screen_fill();
+        break;
     case SCREEN_EVENT_TEXT:
-	screen_text(ev->buf);
-	break;
+        screen_text(ev->buf);
+        break;
     case SCREEN_EVENT_TEXT_RIGHT:
-	screen_text_right(ev->buf);
-	break;
+        screen_text_right(ev->buf);
+        break;
     case SCREEN_EVENT_TEXT_CENTER:
-	screen_text_center(ev->buf);
-	break;
+        screen_text_center(ev->buf);
+        break;
     case SCREEN_EVENT_TEXT_EXTENTS:
-	screen_text_extents(ev->buf);
-	break;
+        screen_text_extents(ev->buf);
+        break;
     case SCREEN_EVENT_TEXT_TRIM:
-	screen_text_trim(ev->buf, ev->payload.d.d1);
-	break;
+        screen_text_trim(ev->buf, ev->payload.d.d1);
+        break;
     case SCREEN_EVENT_CLEAR:
-	screen_clear();
-	break;
+        screen_clear();
+        break;
     case SCREEN_EVENT_CLOSE_PATH:
-	screen_close_path();
-	break;
+        screen_close_path();
+        break;
     case SCREEN_EVENT_EXPORT_PNG:
-	screen_export_png(ev->buf);
-	break;
+        screen_export_png(ev->buf);
+        break;
     case SCREEN_EVENT_DISPLAY_PNG:
-	screen_display_png(ev->buf, ev->payload.bd.d1, ev->payload.bd.d2);
-	break;
+        screen_display_png(ev->buf, ev->payload.bd.d1, ev->payload.bd.d2);
+        break;
     case SCREEN_EVENT_ROTATE:
-	screen_rotate(ev->payload.d.d1);
-	break;
+        screen_rotate(ev->payload.d.d1);
+        break;
     case SCREEN_EVENT_TRANSLATE:
-	screen_translate(ev->payload.d.d1, ev->payload.d.d2);
-	break;
+        screen_translate(ev->payload.d.d1, ev->payload.d.d2);
+        break;
     case SCREEN_EVENT_SET_OPERATOR:
-	screen_set_operator(ev->payload.i.i1);
-	break;
+        screen_set_operator(ev->payload.i.i1);
+        break;
     case SCREEN_EVENT_POKE:
-	screen_poke(ev->payload.bi.i1, ev->payload.bi.i2,
-		    ev->payload.bi.i3, ev->payload.bi.i4,
-		    ev->buf);
-	break;
+        screen_poke(ev->payload.bi.i1, ev->payload.bi.i2, ev->payload.bi.i3, ev->payload.bi.i4, ev->buf);
+        break;
     case SCREEN_EVENT_CURRENT_POINT:
-	screen_current_point();
-	break;
-    default:
-	;;
-    }    
+        screen_current_point();
+        break;
+    default:;

```

Comment:
What's the idea behind a double semi-colon here? (Sorry if this is pedantic)

Reply:
the single semicolon is just an empty statement for the default case. idk, my IDE linter likes it.